pkg/channel/consolidated/reconciler/controller/kafkachannel_test.go

```diff
@@ -750,16 +833,24 @@ func (ca *mockClusterAdmin) DeleteConsumerGroup(group string) error {
 
 var _ sarama.ClusterAdmin = (*mockClusterAdmin)(nil)
 
-func makeDeploymentWithImageAndReplicas(image string, replicas int32) *appsv1.Deployment {
+func makeDeploymentWithImageAndReplicasAndConfigMapHash(image string, replicas int32, configMapHash string) *appsv1.Deployment {
```

Comment:
nit: And..And..And , Oxford folks will scold us :smile: , maybe makeCustomDeployment or something similar?

Reply:
:) Ok, changed it :)